KarhooUISDK/Screens/QuoteList/QuoteListFilters/QuoteFilterHandler.swift

```diff
@@ -51,10 +51,19 @@ class KarhooQuoteFilterHandler: QuoteFilterHandler {
     ) -> [Quote] {
         quotes.filter { quoteToFilter in
             let quoteMeetsFilteringConditions: [Bool] = segregatedFilters.map { filters in
-                let categoryFitlerConditionMet = filters.value.first { filterOfGivenCategory in
-                    filterOfGivenCategory.conditionMet(for: quoteToFilter)
+                
+                switch filters.key {
```

Comment:
Could you please add some documentation on this method for future reference? To explain the difference in logic between the 3 cases on line 56 and the default handling